### ISSUE TITLE
Add ORCID iD to a user's public profile when it is connected to their PhysioNet account. 

### ISSUE THREAD

**Issue by briangow** [MIT-LCP/physionet-build#1234]
As @bemoody suggested, it would be good to add a user's ORCID iD to their public profile https://www.physionet.org/users/<username> if they have linked to ORCID from their PhysioNet profile.  See #870 for related information regarding linking to ORCID accounts.

**Comment by Lucas-Mc** [comment]
Fixed in #1268.

**Comment by tompollard** [comment]
I don't think this issue should have been closed. ORCID iDs are not currently displayed in public profiles (as far as I'm aware).

**Comment by Lucas-Mc** [comment]
I thought they were? I can see mine? Ohhh that "other" public profile

**Comment by briangow** [comment]
Correct, we don't currently display ORCID on a users public profile.  I should have made it clear in the original post that this refers to what is seen at https://physionet.org/users/physionet_username/ .  

We may add this functionality in the future but if so we want to give the user the option to add ORCID to their public profile.  We don't want to automatically do this.

**Comment by tompollard** [comment]
@Lucas-Mc did you mean to close this?

**Comment by Lucas-Mc** [comment]
@tompollard Yes since we are still unsure as the best way to add it while respecting the interests of the user... I'm planning on opening a new one about enhancing the private profiles before that though
